ui/src/Components/ManagerPane.js

```diff
@@ -50,7 +57,7 @@ const ManagerPane = props => {
       </nav>
       <div className="child-content">
         {/* need to remove feedback and update using props */}
-        <Feedback />
+        {FeedbackView(selectedIndex)}
```

Comment:
function names must always be in lowercase

Reply:
done